### PR TITLE
docs(Image): Moved over to use ExampleConfig

### PR DESCRIPTION
**Pull request by HHogg**

### CODE REVIEW

File: src/components/image/ImageCircle.js [hunk 1]
```diff
@@ -6,13 +6,21 @@ import './ImageCircle.css';
 
 export default class ImageCircle extends Component {
   static propTypes = {
+    /** Border applied around the image */
     border: PropTypes.oneOf(['small', 'large']),
+    /** Fallback content when the image fails to load */
     children: PropTypes.node,
+    /** Name of the icon to added to the hover overlay */
```
Comment (thatguynamedandy) [inline src/components/image/ImageCircle.js]: Slight language malfunction here and the one below. `to be added`?

File: src/components/image/ImageCircle.js [hunk 2]
```diff
@@ -6,13 +6,21 @@ import './ImageCircle.css';
...
     overlayIconName: PropTypes.string,
+    /** Size of the icon to added to the hover overlay */
     overlayIconSize: PropTypes.string,
+    /** Size (with unit) that will applied to the width and height of the image */
```
Comment (thatguynamedandy) [inline src/components/image/ImageCircle.js]: Another missing `be` here too I think 🐝

File: src/components/image/ImageCircle.js [hunk 3]
```diff
@@ -6,13 +6,21 @@ import './ImageCircle.css';
...
   static propTypes = {
+    /** Border applied around the image */
     border: PropTypes.oneOf(['small', 'large']),
+    /** Fallback content when the image fails to load */
     children: PropTypes.node,
+    /** Name of the icon to be added to the hover overlay */
     overlayIconName: PropTypes.string,
+    /** Size of the icon to be added to the hover overlay */
     overlayIconSize: PropTypes.string,
+    /** Size (with unit) that will be applied to the width and height of the image */
```
Comment (pmsorhaindo) [inline src/components/image/ImageCircle.js]: Super picky but, when adding `(with unit)` it implies that the `overlayIconSize` above that doesn't have `(with unit)` in its explanation doesn't take a unit. I'm not 100% sure but I think it's the case that all string properties named size take a number with unit, so I think this parenthesis should be everywhere or nowhere and just implied.

**Comment by pmsorhaindo** [comment]
Small super not a big deal comment but I approve.

File: src/components/image/ImageCircle.js (hunk 3 above)
Comment (HHogg) [inline src/components/image/ImageCircle.js]: Yeah good point... I agree it should just be implied as it's the general pattern in Axiom. I'll remove them 👍

**Pull request merged**